plus/src/main/java/org/apache/calcite/adapter/os/SqlShell.java

```diff
@@ -118,17 +118,18 @@ private static String model() {
   /** Main entry point. */
   @SuppressWarnings("CatchAndPrintStackTrace")
   public static void main(String[] args) {
-    try (PrintWriter err =
-             new PrintWriter(
-                 new OutputStreamWriter(System.err, StandardCharsets.UTF_8));
-         InputStreamReader in =
-             new InputStreamReader(System.in, StandardCharsets.UTF_8);
-         PrintWriter out =
-             new PrintWriter(
-                 new OutputStreamWriter(System.out, StandardCharsets.UTF_8))) {
+    try {
+      final PrintWriter err =
+          new PrintWriter(
+              new OutputStreamWriter(System.err, StandardCharsets.UTF_8));
```

Comment:
Previously, `try-with-resource` syntax will close stream, 
but now we haven't closed them.

Should we close them?

Reply:
I think it's ok to not close them, we only need to ensure they flush all the data into enclosed stream (for now, only `out` is used, and it's flushed in `finally` block).